index.src.html

```diff
@@ -1079,37 +1081,50 @@ <h3 id="deprecation-report">Deprecation</h3>
 
   Deprecation reports have the <a>report type</a> "deprecation".
 
-  A deprecation report's [=report/body=] contains the following fields:
+  Deprecation reports are <a>observable from JavaScript</a>.
 
-    - <dfn for="Deprecation">id</dfn>: an implementation-defined string
-      identifying the feature or API that will be removed. This string can be
-      used for grouping and counting related reports.
+  <pre class="idl">
+    interface DeprecationReportBody : ReportBody {
+      readonly attribute DOMString id;
+      readonly attribute Date? anticipatedRemoval;
+      readonly attribute DOMString message;
+      readonly attribute DOMString? sourceFile;
+      readonly attribute long? lineNumber;
+      readonly attribute long? columnNumber;
+    };
+  </pre>
 
-    - <dfn for="Deprecation">anticipated_removal</dfn>: A date indicating
-      roughly when the browser version without the specified API will be
-      generally available (excluding "beta" or other pre-release channels). This
-      value should be used to sort or prioritize warnings. If unknown, this
-      field should be set to null, and the deprecation should be considered low
-      priority (removal may not actually occur).
+  A deprecation report's [=report/body=], represented in JavaScript by
+  {{DeprecationReportBody}}, contains the following fields:
 
-    - <dfn for="Deprecation">message</dfn>: A human-readable string with
-      details typically matching what would be displayed on the developer
-      console. The message is not guaranteed to be unique for a given
-      [=Deprecation/id=] (eg. it may contain additional context on how the API
-      was used).
+    - <dfn for="DeprecationReportBody">id</dfn>: an implementation-defined
+      string identifying the feature or API that will be removed. This string
+      can be used for grouping and counting related reports.
 
-    - <dfn for="Deprecation">source_file</dfn>: If known, the file which first
-      used the indicated API, or null otherwise.
+    - <dfn for="DeprecationReportBody">anticipated_removal</dfn>: A
+      JavaScript <a>Date object</a> indicating roughly when the browser version
```

Comment:
Is it worth adding some text about how the `Date` object is rendered in JSON?  (ISO 8601 string?)